rsmtool/analyzer.py

```diff
@@ -610,7 +638,7 @@ def metrics_helper(human_scores,
             - `wtkappa`:  quadratic weighted kappa
             - `exact_agr`: exact agreement
             - `adj_agr`: adjacent agreement with tolerance set to 1
-            - `SMD`: standardized mean difference
+            - `SMD`: standardized mean difference (or `DSM`).
```

Comment:
"- One of the following:
     * SMD: standardized... if use_diff_std_means is false
     * DSM: difference between standardized means if use_diff_std_means is true"